shell/test/specs/demo-tests.js

```diff
@@ -463,7 +463,7 @@ describe('Arcs demos', function() {
     // });
   });
 
-  it('can buy gifts', /** @this Context */ function() {
+  it.skip('can buy gifts', /** @this Context */ function() {
```

Comment:
Is it ok to just skip these two tests or should we file issues?

Reply:
Both :) ... will open discussion.